app/rtc_s1_parallel.py

```diff
@@ -0,0 +1,887 @@
+        burst = burst_pol_dict[pol_list[0]]
+
+        flag_bursts_files_are_temporary = (not save_bursts or
+                                           save_imagery_as_hdf5)
+        flag_bursts_secondary_files_are_temporary = (
+            not save_bursts or save_secondary_layers_as_hdf5)
+
+        burst_scratch_path = f'{scratch_path}/{burst_id}/'
+        
+        if not save_bursts:
+            # burst files are saved in scratch dir
+            bursts_output_dir = burst_scratch_path
+        else:
+            # burst files (individual or HDF5) are saved in burst_id dir 
+            bursts_output_dir = os.path.join(output_dir, burst_id)
+            os.makedirs(bursts_output_dir, exist_ok=True)
+        
+        geogrid = cfg.geogrids[burst_id]
+
+        # snap coordinates
+        x_snap = geogrid.spacing_x
+        y_snap = geogrid.spacing_y
+        geogrid.start_x = rtc_s1.snap_coord(geogrid.start_x, x_snap, np.floor)
+        geogrid.start_y = rtc_s1.snap_coord(geogrid.start_y, y_snap, np.ceil)
+
+        # update mosaic boundaries
+        rtc_s1._update_mosaic_boundaries(mosaic_geogrid_dict, geogrid)
+
+        logger.info(f'    reading burst SLCs')
+        radar_grid = burst.as_isce3_radargrid()
+
+        # native_doppler = burst.doppler.lut2d
+        orbit = burst.orbit
+        if 'orbit' not in mosaic_geogrid_dict.keys():
+            mosaic_geogrid_dict['orbit'] = orbit
+        if 'wavelength' not in mosaic_geogrid_dict.keys():
+            mosaic_geogrid_dict['wavelength'] = burst.wavelength
+        if 'lookside' not in mosaic_geogrid_dict.keys():
+            mosaic_geogrid_dict['lookside'] = radar_grid.lookside
+
+        input_file_list = []
+
+        for pol, burst_pol in burst_pol_dict.items():
+            temp_slc_path = \
+                f'{burst_scratch_path}/rslc_{pol}.vrt'
+            temp_slc_corrected_path = (
+                f'{burst_scratch_path}/rslc_{pol}_corrected.{imagery_extension}')
+            
+            if (flag_apply_thermal_noise_correction or
+                    flag_apply_abs_rad_correction):
+                input_burst_filename = temp_slc_corrected_path
+                temp_files_list.append(temp_slc_corrected_path)
+            else:
+                input_burst_filename = temp_slc_path
+
+            temp_files_list.append(temp_slc_path)
+            input_file_list.append(input_burst_filename)
+
+        # At this point, burst imagery files are always temporary
+        geo_burst_filename = \
+            f'{burst_scratch_path}/{product_prefix}.{imagery_extension}'
+        temp_files_list.append(geo_burst_filename)
+
+        # Generate output geocoded burst raster        
+        if save_nlooks:
+            nlooks_file = (f'{bursts_output_dir}/{product_prefix}'
+                           f'_nlooks.{imagery_extension}')
+
+            #if skip_burst_process:
+            os.rename(nlooks_file.replace(output_dir, scratch_path), nlooks_file)
```

Comment:
What does this line do?

Reply:
- The comment `#if skip_burst_process:` is not relevant in this function. I'll remove this
- This is to move the nooks file, which is parent process' scratch directory (i.e. output directory of the child) into the parent process' output directory when `save_nlooks` is turned on.